Training/2.0 Libraries.md

```diff
@@ -24,16 +24,25 @@ When to use which? Each of these libraries has very useful functionality but you
 Make a two-dimensional numpy array. First row will be: [1,2,3] and second row will be [4,5,6]. 
 Using numpy functionality find the mean, minimum and max value of your array. Index the maximum value and assign it to a variable named max. Index the minimum value and assign it to a variable named min. 
 
-```
+``` np2d = np.array([[1,2,3][4,5,6]])
+``` max = (np2d.max())
+``` min = (np2d.min())
+``` average = (np2d.mean())
+``` max = 6
+``` min = 1
+``` average = 3.5
 
 ```
 
```

Comment:
Remember to put a comma between the two sets of numbers in the array: [1,2,3],[4,5,6] - if not, it will make it a tuple
The rest is correct!